src/slic3r/GUI/OptionsGroup.hpp

```diff
@@ -155,6 +156,14 @@ class OptionsGroup {
 							if (m_fields.find(id) == m_fields.end()) return nullptr;
 							return m_fields.at(id).get();
     }
+
+    inline Line*	get_line(const t_config_option_key& id) {
+		for (Line& line : m_lines)
+			if (line.has_only_option(id))
```

Comment:
Shall we is_separator() too?
I can't recall the exact causes why I added that check. I remember there were crashes caused by separators

Reply:
ill look back into the commit it was added and get back to you

Reply:
The commit is from the implementation of internal bridge speed. I didn't get any crashes, but that doesn't mean much. I am kinda leaning towards just going back the the original BBL function...

Reply:
hehehe, that change was introduced by myself.
It was crashing when a separator line was added